analyzers/tests/SonarAnalyzer.UnitTest/TestCases/OperatorsShouldBeOverloadedConsistently.CSharp9.cs

```diff
@@ -1,15 +1,33 @@
-﻿public record Compliant
+﻿public record Compliant // Compliant as == operator cannot be overloaded for records.
 {
     public static object operator +(Compliant a, Compliant b) => new object();
     public static object operator -(Compliant a, Compliant b) => new object();
+    public static object operator *(Compliant a, Compliant b) => new object();
+    public static object operator /(Compliant a, Compliant b) => new object();
+    public static object operator %(Compliant a, Compliant b) => new object();
 }
 
-public record OnlyPlus // Compliant - FN, should implement `-` too
+public record OnlyPlus // Compliant
 {
     public static object operator +(OnlyPlus a, OnlyPlus b) => new object();
```

Comment:
Don't we want to suggest `-` in this case? `==` cannot be overloaded but `-` can.

Reply:
Before these modifications the rule suggested to implement -, but that implementation was not aligned with the rule description. So, I changed the rule to suggest just overloading the == operator as the rule originally suggested. I am open to any discussion if you think we should go back to the previous implementation.